package.json

```diff
@@ -3,7 +3,7 @@
 	"version": "0.0.1",
 	"type": "module",
 	"scripts": {
-		"dev": "bun database:up && bun prisma:push && vite dev",
+		"dev": "bun database:up && bun prisma:generate && bun prisma:push && vite dev",
```

Comment:
Add `--watch` to generate

Reply:
If you add watch the next command won't run, will need to use the package concurrency to make it work

Reply:
It might make more sense to move prisma:push to database:up since its a one-shot and you need to run it manually when you make DB changes